test/cmake/threads/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(threads VERSION 0.42)
```

Comment:
Where is this version number from?

Reply:
This is the version of number of the project being built.    I copied this from on of our other tests.  I don't think it matters for the purposes of the test.

Reply:
Oh, isn't it the version number of the "threads" library? I thought this mechanism lets you ask for a library of a version.

Reply:
Ah, nevermind, I misread this.. this isn't a require line.